src/test/java/Calculator/CalculatorTest.java

```diff
@@ -0,0 +1,124 @@
+package Calculator;
+
+import Calculator.exception.ImpossibleComputeExpException;
+import Calculator.exception.OnlyNumberException;
+import Calculator.exception.OnlyOperatorException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CalculatorTest {
+
+    @BeforeEach
+    public void setup() {
+        ExpressionGenerator.isOnlyNumberExp = true;
+        ExpressionGenerator.isOnlyOperatorExp = true;
+    }
+
+    @Test
+    @DisplayName("null 또는 빈 문자열일 경우 예외처리 테스트")
+    public void validateTest() {
+
+        assertThatIllegalArgumentException().isThrownBy(() -> {
+            ExpressionGenerator.validateAndSplitExpression(null);
+        });
+
+        assertThatIllegalArgumentException().isThrownBy(() -> {
+            ExpressionGenerator.validateAndSplitExpression("");
+        });
+    }
```

Comment:
중복되는 테스트는 @ParameterizedTest를 사용해주세요.
null과 빈 문자열은 @NullAndEmptySource 사용도 가능합니다~

개인적으로는 테스트 메소드에 `~ 하는지 테스트` 처럼 테스트란 단어를 굳이 사용할 필요는 없는 것 같아요
**'~인지 테스트'** 보단 **'~일 때 예외를 던진다'** 와 같은 설명이 어떤 테스트 파악인지 쉽다고 생각해요 :)


```suggestion
    @ParameterizedTest(name = "expression: {0}")
    @NullAndEmptySource
    @DisplayName("null 또는 빈 문자열인 경우 예외를 던진다.")
    void validateTest(String expression) {
        assertThatIllegalArgumentException()
                .isThrownBy(() -> ExpressionGenerator.validateAndSplitExpression(expression));
    }
```